config/routes.rb

```diff
@@ -1,7 +1,14 @@
 Rails.application.routes.draw do
+  mount_devise_token_auth_for 'User', at: 'api/v1/auth', skip: [:omniauth_callbacks]
   namespace :api do
+
     namespace :v0 do
```

Comment:
are these namespaces necessary? and do you really want a pings controller here?

Reply:
We decided to remove the pings controllers line + test + controller.